export const de = {
  app: {
    title: 'Worklogs',
  },

  routes: {
    home: 'Ãœbericht',
    invoice: 'Rechnungen',
    about: 'Technische Informationen',
    settings: 'Settings',
    signIn: 'Anmelden',
  },
  errors: {
    restart: 'Neustart',
    reloadPage: 'Seite neu laden',
    unexpectedError:
      'Es ist ein unerwarteter Fehler aufgetreten. Bitte versuche, die Seite neu zu laden',
    componentStack: 'Entwicklerinformationen (bitte an Entwickler weiterleiten)',
    errorName: 'Fehlername',
    errorMessage: 'Nachricht',
    unknownError: 'Unbekannter Fehler',
    unexpectedErrorDialog:
      'Ein unerwarteter Fehler ist aufgetreten. Bitte laden Sie die Seite neu. Falls das Problem weiterhin besteht, wenden Sie sich an den technischen Dienst',
  },
  about: {
    buildTime: 'Build Datum',
    gitRef: 'Git Ref',
    gitBranch: 'Git Branch',
  },
}
